### ISSUE TITLE
Group block: Nested blocks in wide alignment break

### ISSUE THREAD

**Issue by davipontesblog** [Automattic/wp-calypso#39588]
**Important: This does not seem to affect Varia-child themes, or Twenty-twenty, which leads me to believe after lots of testing that it has to do with the theme's lack of block editor styles for handling the nesting of wide-aligned blocks inside the Group block. I tested nesting the same below described blocks and reproducing the same steps as described here with the Cover block instead of the Group block, and didn't find any issues.**

### Scope
This seems to affect and happen with any block set to Wide alignment nested within a Group Block, as I tested the same alignment (Wide) for the following blocks inside a Group block in full-width alignment alongside the problematic Columns block (below it), and the same issue happens either in editor or in live preview, or both:
- Media & Text
- Gallery
For the purpose of this report and the steps to reproduce, I am using Columns nested inside a Group block, but other nested blocks break too as mentioned above. 

See below:

#### Steps to reproduce
Note: make sure the site is not using a theme that has block editor styles applied, like any Varia-child themes, or Twenty Twenty for example.
1. Start from URL: https://wordpress.com/pages/domain
2. Add a Group block, make it full or wide width.
3. Nest inside a Columns Block, **make this wide width**.
4. Check the elements in the nested block, and also in the live page preview after updating the page in the editor.

#### What I expected
To see the elements displayed in wide alignment inside the Group Block.

#### What happened instead
This will break the Columns block, making it break either in the editor, the live preview, or both, depending on the theme. It makes the alignment broken where content is positioned outside of the viewpoint on the window.

#### Environment
MacOS Catalina, Google Chrome Version 80.0.3987.106 (Official Build) (64-bit)
Test site: test511527546.wordpress.com, page https://wordpress.com/block-editor/page/test511527546.wordpress.com/46
_Note: multiple other reports in related GH issues listed below:_
I believe this might be the root cause or at the very least related to the following issues (which include user reports):
https://github.com/Automattic/themes/issues/1711
https://github.com/Automattic/themes/issues/1683
https://github.com/Automattic/themes/issues/980
https://github.com/Automattic/themes/issues/670

#### Screenshot / Video
- Video showing the issue on Radcliffe 2 as described in the "Steps to Reproduce" section: https://d.pr/v/QdNWhY/yhV6XLduJJ
- Video recording showing the issue also happening with other nested blocks in wide-alignment setting nested within a Group block: https://d.pr/v/moZc0O/UtdprBI5H2
The same happens on Business, and Twenty Nineteen, and I suspect it affects more themes if we test more.
Business theme:
<img width="1305" alt="Business preview" src="https://user-images.githubusercontent.com/31009008/75003096-a87cb880-5434-11ea-8a74-2c2fbc6db993.png">
Radcliffe 2 theme:
<img width="1305" alt="Radcliffe 2 preview" src="https://user-images.githubusercontent.com/31009008/75003115-b6cad480-5434-11ea-94ef-af74798e05cf.png">
Twenty-Nineteen theme:
<img width="1307" alt="Twenty Nineteen editor" src="https://user-images.githubusercontent.com/31009008/75003132-c3e7c380-5434-11ea-8061-a213f7e4ce31.png">

#### Context / Source
#painpoint #manual-testing #user-report

**Comment by davipontesblog** [comment]
The solution that works to fix this, in any case scenario we have seen so far, is to change the nested block alignment to full-width as opposed to wide-aligned.

**Comment by jonnyruzek** [comment]
Karuna: 18830495-hc

**Comment by chad1008** [comment]
Same issue on Ixion (using Hever home page template)
2811518-zen

**Comment by kjellr** [comment]
> Twenty-Nineteen theme

@allancole When you have a moment, can you sync WP.com's version of Twenty Nineteen with the one on WP.org `trunk`? That should fix it for that theme.

**Comment by fylgjur** [comment]
@davipontesblog , I actually found the opposite fix in https://github.com/Automattic/themes/issues/2160 to yours above. Where the Group blocks were already Full Width, I found it fixed the issue to change them to Wide Width in my tests...

**Comment by ianstewart** [comment]
> Important: This does not seem to affect Varia-child themes, or Twenty-twenty, which leads me to believe after lots of testing that it has to do with the theme's lack of block editor styles for handling the nesting of wide-aligned blocks inside the Group block.

@davipontesblog is this still reproducible and if so should it be moved to the Themes repo for maintenance?

**Comment by davipontesblog** [comment]
I just tested this on a site with the Hever theme (Varia child theme) and it is happening in the Editor, but not on the front end:

https://user-images.githubusercontent.com/31009008/117127887-12039100-ad73-11eb-9119-9886137eaa8a.mov

It happens with the Columns block here, in both Wide and Full-Width alignments.

**Comment by cuemarie** [comment]
_This report was found during a recent backlog sweep._

📌 SCRUBBING : RESULT - _Replicated / Could Not Replicate / Uncertain_
- Tested on Simple - **Replicated**

📌 FINDINGS/SCREENSHOTS/VIDEO
- I can still replicate this, tested using Business theme on Simple site.

With both a Columns block and a Media & Text block set to `Wide Width`, inside a Group block set to `Full Width`, the following CSS gets applied to the child blocks:

```
  @media only screen and (min-width: 815px)
.entry-content .alignwide {
  margin-left: calc(-1*(1235px - calc(7*(80px + 25px) - 25px))/2);
  margin-left: calc(-1*(1235px - var(--grid-column-7))/2);
}
```

https://github.com/Automattic/wp-calypso/assets/27249804/5fa92def-cbbf-4ecb-b14a-b57881898a48

📌 ACTIONS
- Assigned to Themes
